src/app/AppInitGate.tsx

```diff
@@ -41,9 +41,9 @@ const AppInitGate = ({ loading, children }: Props) => {
 
   const initResult = useAsync(
     async () => {
-      Logger.debug(TAG, 'Starting init')
-      await Promise.all([i18nInitializer(), ValoraAnalytics.init()])
-      Logger.debug(TAG, 'init completed')
+      Logger.debug(TAG, 'Starting AppInitGate init')
+      await Promise.all([i18nInitializer(), waitUntilSagasFinishLoading()])
+      Logger.debug(TAG, 'AppInitGate init completed')
```

Comment:
Nothing to do in this PR, but this reminded me we should measure how long it takes from app start to [dimissing the loading splash screen](https://github.com/valora-inc/wallet/blob/6d076bc9b2a83fa340941a8a1a241af8e5180382/src/navigator/Navigator.tsx#L643).